### ISSUE TITLE
Fatal Python error: Aborted

### ISSUE THREAD

**Issue by lhq15606085117** [ultralytics/yolov5#12655]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Thread 0x000047b8 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x000005e8 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00004350 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00000f48 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00001638 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x000034b8 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00001fac (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x000043f0 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x000001d0 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x000039c0 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00002028 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00004620 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x0000130c (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00001068 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00002f68 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00002780 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x000043a8 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x0000150c (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00001a94 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00002768 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 320 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\multiprocessing\queues.py", line 231 in _feed
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 975 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Thread 0x00004734 (most recent call first):
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 324 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 622 in wait
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\site-packages\tqdm\_monitor.py", line 60 in run
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 1038 in _bootstrap_inner
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\threading.py", line 995 in _bootstrap

Main thread:
Current thread 0x000033f0 (most recent call first):
  File "<__array_function__ internals>", line 200 in dot
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\transforms.py", line 2436 in get_affine
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\transforms.py", line 2409 in transform_affine
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\_api\deprecation.py", line 297 in wrapper
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\transforms.py", line 1495 in transform
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\transforms.py", line 468 in transformed
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\axis.py", line 2515 in get_tick_space
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\ticker.py", line 2086 in _raw_ticks
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\ticker.py", line 2148 in tick_values
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\ticker.py", line 2140 in __call__
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\axis.py", line 1495 in get_majorticklocs
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\axis.py", line 1275 in _update_ticks
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\axis.py", line 1424 in get_majorticklabels
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\axis.py", line 1467 in get_ticklabels
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\matplotlib\axes\_base.py", line 73 in wrapper
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\seaborn\axisgrid.py", line 1346 in __init__
  File "C:\Users\12920\AppData\Roaming\Python\Python311\site-packages\seaborn\axisgrid.py", line 2119 in pairplot
  File "D:\web_ruanjian\py_ruanjian\yolov5\yolov5-master\utils\plots.py", line 305 in plot_labels
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\contextlib.py", line 81 in inner
  File "D:\web_ruanjian\py_ruanjian\yolov5\yolov5-master\utils\loggers\__init__.py", line 175 in on_pretrain_routine_end
  File "D:\web_ruanjian\py_ruanjian\yolov5\yolov5-master\utils\callbacks.py", line 73 in run
  File "d:\web_ruanjian\py_ruanjian\yolov5\yolov5-master\train.py", line 292 in train
  File "d:\web_ruanjian\py_ruanjian\yolov5\yolov5-master\train.py", line 616 in main
  File "d:\web_ruanjian\py_ruanjian\yolov5\yolov5-master\train.py", line 836 in <module>
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\site-packages\spyder_kernels\py3compat.py", line 356 in compat_exec
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 473 in exec_code
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 615 in _exec_file
  File "D:\web_ruanjian\py_ruanjian\Anaconda\Lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 528 in runfile
  File "C:\Users\12920\AppData\Local\Temp\ipykernel_1368\575683251.py", line 1 in <module>

Restarting kernel...

**Comment by glenn-jocher** [comment]
@lhq15606085117 hello! It looks like you're experiencing a crash during training, which can be caused by various issues. Here are a few steps you can take to troubleshoot the problem:

1. **Check your environment**: Ensure that you have the correct versions of PyTorch, CUDA, and other dependencies installed. Compatibility issues can sometimes lead to crashes.

2. **Reduce resource usage**: Try reducing the batch size or using a simpler model to see if the issue is related to system resource limits.

3. **Update your code**: Make sure you're using the latest version of YOLOv5 from the repository, as updates often include bug fixes.

4. **Isolate the issue**: If possible, try to run the code on a different machine or environment to determine if the issue is specific to your current setup.

5. **Inspect the logs**: Look for any error messages or warnings that occur before the crash, as they can provide clues about the underlying issue.

6. **Check for known issues**: Review the YOLOv5 GitHub issues and discussions again for any similar reports, as there might be solutions or workarounds provided by the community.

If you continue to face difficulties, please provide more details about your setup, including the versions of the relevant libraries you're using, and the exact command or script you're running. This information will help us diagnose the problem more effectively.

For further guidance, you can refer to our documentation at https://docs.ultralytics.com/yolov5/.

Thank you for reaching out, and we hope to get you up and running soon! 🚀
